processing/src/main/java/io/druid/query/groupby/epinephelinae/LimitedBufferHashGrouper.java

```diff
@@ -453,7 +453,7 @@ public AlternatingByteBufferHashTable(
       subHashTable2Buffer.limit(tableArenaSize);
       subHashTable2Buffer = subHashTable2Buffer.slice();
 
-      subHashTableBuffers = new ByteBuffer[] {subHashTable1Buffer, subHashTable2Buffer};
+      subHashTableBuffers = new ByteBuffer[]{subHashTable1Buffer, subHashTable2Buffer};
```

Comment:
Could you add a checkstyle rule (I think it could be a simple regex) to catch such things automatically?

Reply:
Outside scope of this PR